xcp_d/utils/plot.py

```diff
@@ -1006,3 +1006,133 @@ def _get_tr(img):
     except AttributeError:  # Error out if not in cifti
         return img.header.get_zooms()[-1]
     raise RuntimeError("Could not extract TR - unknown data structure type")
+
+
+def plot_alff_reho_volumetric(output_path, filename, bold_file):
+    """Plot ALFF/ReHo mosaic plot for Niftis."""
+    from bids.layout import parse_file_entities
+    from nilearn import plotting as plott
+    from templateflow.api import get as get_template
+
+    space = str(parse_file_entities(bold_file)["space"])
+    try:
+        resolution = str(parse_file_entities(bold_file)["res"])
+    except Exception:
+        resolution = 1
```

Comment:
```suggestion
    resolution = parse_file_entities(bold_file).get("res", "1")
```

The output from `parse_file_entities` should be a dictionary, so you can just use `.get()`, which will return None by default if the key isn't present in the dictionary, or you can set the value it will return via the second parameter (e.g., `.get(key, default_value)`).